test/escaping.test.js

```diff
@@ -19,7 +19,7 @@ describe('escaping', () => {
 
     expect(getMetaInfo(component, [[/&/g, '&amp;']])).toEqual({
       title: 'Hello &amp; Goodbye',
-      titleChunk: 'Hello &amp; Goodbye',
+      titleChunk: 'Hello & Goodbye',
```

Comment:
I guess that change is applied because escaping ampersands isn't needed for the document title, right?

Reply:
Good catch, that might still be an issue actually (on client side). Will check that.

This changes comes from that I now check if the metaInfo key is a configuration option key as defined here: https://github.com/nuxt/vue-meta/blob/next/src/shared/constants.js#L30

Eg `titleChunk` is never used directly in html, its always applied to a template and only the result after applying the template should be escaped.

Reply:
I checked and this is not an issue client side because it replaced with unicode chars and not with html special chars